yarn.lock

```diff
@@ -9118,6 +9118,19 @@ cli-cursor@^3.1.0:
   dependencies:
     restore-cursor "^3.1.0"
 
+<<<<<<< HEAD
```

Comment:
`yarn.lock` was committed with merge conflicts which caused the CI checks to fail instantly.
There are a few prettier/linter issues (most of them can be fixed with `yarn workspace @joystream/cli format`)

Reply:
done here https://github.com/Joystream/joystream/pull/2258/commits/91fa8e99eb0d25e4f1f2b2bbb6b504860634bfc3

Reply:
Looks like this one wasn't comitted?

Reply:
Done https://github.com/Joystream/joystream/pull/2258/commits/2db5a7d3e8fe9d2e50aa90b0febe4be3b8dd03f2